common/djangoapps/track/utils.py

```diff
@@ -1,7 +1,7 @@
 """Utility functions and classes for track backends"""
 
-from datetime import datetime, date
 import json
+from datetime import datetime, date
```

Comment:
Tiny nit: I don't understand why this moved - do we try to keep `import YYY` statements above `from XXX import ZZZ` ones?